content/intro-to-storybook/ember/en/get-started.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Storybook for Ember tutorial'
+tocTitle: 'Get started'
+description: 'Setup Storybook in your development environment'
+---
+
+Storybook runs alongside your app in development mode. It helps you build UI components isolated from the business logic and context of your app. This edition of Learn Storybook is for Ember; other editions exist for [React](/react/en/get-started), [React Native](/react-native/en/get-started), [Vue](/vue/en/get-started), [Angular](/angular/en/get-started) and [Svelte](/svelte/en/get-started).
+
+![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
+
+## Setup Ember Storybook
+
+We’ll need to follow a few steps to get the build process set up in your environment. To start with, we want to use [ember-cli](https://github.com/ember-cli/ember-cli) to setup our build system, and enable [Storybook](https://storybook.js.org/) and [Qunit](https://qunitjs.com/) testing in our app. Let’s run the following commands:
+
+```bash
+# Create our application:
+ember new taskbox
+
+cd taskbox
+
+# Add Storybook:
+npx -p @storybook/cli sb init
+
+# Add Ember storybook adapter
+ember install @storybook/ember-cli-storybook
+```
+
+<div class="aside">
+<p>
+At the time of writing this tutorial adding the @storybook/ember-cli-storybook, will yield the following message :
+</p>
+The <code>`ember generate <entity-name>`</code> command requires an entity name to be specified. <br/>For more details, use `ember help`.
+<p>Don't worry about it. Everything should be installed and properly configured.</p>
+</div>
+
+We'll need to install one additional package in our app, this one will help us out shortly. Run the following command:
+
+```bash
+ember install ember-truth-helpers
+```
+
+And finally make a small change to our <code>package.json</code> to allow Storybook to work properly with our Ember app.
+
+Add the following entry to your scripts:
+
+```javascript
+// package.json
+{
+    "scripts":{
+        "storybook-dev":"npm-run-all --aggregate-output --continue-on-error --parallel start storybook",
+        "pre-build":"ember build",
+        "deploy-storybook":"run-s pre-build build-storybook"
```

Comment:
@tmeasday do you mean the `deploy-storybook`? If that's the case the reason why that script is in this section, is two-pronged. 
One, it was taken from the deploy section to this section. I'm aiming for having almost of the setup done in this section where it should and when he/she reaches that section he'll be only concerned with the console commands and setup the repo in github/netlify so that he/she can get his/hers storybook deployed with the least ammount of work possible and with that keep the section a bit more streamlined. 
Two, this indirectly ties into the issue i've raised recently in the storybook-chromatic repo. The same goes for this, as the reader is going through it is using the dev server to check his/hers Storybook working locally. When he reaches that section if he just issues `build-storybook` and pushes it to netlify is in for a surprise (pardon the bad pun), he'll get into the same problems with the live-reload file. From my testing it seems that ember in conjunction with Storybook and probably other frameworks aswell is highly dependant of the build and the content of what's inside the `dist` folder, so having this works as a preventive measure against  possible issues that could arise that are technically related to Storybook itself

Reply:
with the latest commit this is now solved i believe. The proposed change above takes care of this. Also the image for the netlify settings is now fixed to use the npm one and the one related to ember was removed.